<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_BLOCK</class>
 <widget class="QWidget" name="Widget_BLOCK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="DZ_time">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_current">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>电流:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DZ_grade">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>6.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>采集完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cycle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>周期:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>更换电机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pwr">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>功率:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="waveLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>采集</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dzTab">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>491</width>
     <height>107</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电压</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电流</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>功率</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_grade_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>档位：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_grade_10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>98</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border:0px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>130</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_volt">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>外部电压：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="BLOCK_Volt">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_fre">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>频率：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="BLOCK_Fre">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>99.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_channel">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>通道：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="BLOCK_channel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>41</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>3.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_hide">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DZ_Cycle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>3.000000000000000</double>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>BLOCK_Volt</tabstop>
  <tabstop>DZ_time</tabstop>
  <tabstop>DZ_grade</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>BLOCK_Fre</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
